tmc-langs/src/lib.rs

```diff
@@ -520,32 +520,42 @@ pub fn init_tmc_client_with_credentials(
 // TODO: parallel downloads
 pub fn update_exercises(
     client: &TmcClient,
-    client_name: &str,
+    projects_dir: &Path,
 ) -> Result<DownloadOrUpdateCourseExercisesResult, LangsError> {
-    log::debug!("updating exercises for {}", client_name);
+    log::debug!("updating exercises in {}", &projects_dir.display());
```

Comment:
The & here is unnecessary

Reply:
removed debug prints and fixed the unwrap